docs/guide-tequila-api-key.md

```diff
@@ -1,6 +1,20 @@
 ---
-title: "Guide: Get your Tequila API key"
-sidebar_label: "Guide: Get your Tequila API key"
+title: Guide: Get your Tequila API key
+sidebar_label: Guide: Get your Tequila API key
 ---
 
-_This page is a stub._
+For successful run our the GraphQL proxy server, you need to has set the Tequila **API key** to `API_KEY` environment variable.
+
+To get your **API key** you will need an account at the [Tequila](tequila.kiwi.com)
+. You can register and log in on the tequila.kiwi.com page.
+
+After successfully signing in to your account, you can find out what **Tequila** is about, how to work directly with the **Tequila API**.
+
+To get your **API key** for the GraphQL got into the My applications page.
+Create the new app, by clicking on the `add application` button.
+
+Then you should choose your partnership the **<span>Kiwi</span>.com Affiliate Program** or **Book with <span>Kiwi</span>.com**. More information about partnership types, you will find there.
```

Comment:
Why the `<span>` tags?

> More information about partnership types, you will find there.
Where? I'd probably remove this sentence if the info is available on the page.

Reply:
Because without this "hack", it behaves like a link

Reply:
done